ios/browser/api/favicon/favicon_driver.mm

```diff
@@ -154,7 +154,8 @@ - (void)webView:(WKWebView*)webView
   if (!urls.empty()) {
     driver_observer_ = std::make_unique<FaviconDriverObserver>(
         base::BindRepeating(^(const GURL& icon_url, const gfx::Image& icon) {
-          callback(net::NSURLWithGURL(icon_url), icon.ToUIImage());
+          callback(net::NSURLWithGURL(icon_url),
+                   icon.IsEmpty() ? nullptr : icon.ToUIImage());
```

Comment:
Is it OK to pass `nullptr` here? Not sure where to follow to see if this callback is defined publicly anywhere and needs a `nullable` annotation. Is this the callback that eventually gets to `web_image.h`'s method?

Reply:
Yeah it's okay to be null on the iOS side :)